src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/health/HealthInfo.kt

```diff
@@ -2,15 +2,29 @@ package uk.gov.justice.digital.hmpps.personrecord.health
 
 import org.springframework.boot.actuate.health.Health
 import org.springframework.boot.actuate.health.HealthIndicator
+import org.springframework.boot.actuate.health.Status
 import org.springframework.boot.info.BuildProperties
 import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.personrecord.client.MatchScoreClient
 
 /**
  * Adds version data to the /health endpoint. This is called by the UI to display API details
  */
+
 @Component
-class HealthInfo(buildProperties: BuildProperties) : HealthIndicator {
+class HealthInfo(buildProperties: BuildProperties, val matchScoreClient: MatchScoreClient) : HealthIndicator {
   private val version: String = buildProperties.version
 
-  override fun health(): Health = Health.up().withDetail("version", version).build()
+  override fun health(): Health {
+    return try {
+      val healthStatus = matchScoreClient.getMatchHealth()
+      return when (healthStatus?.status) {
+        Status.UP.toString() -> Health.up().withDetail("version", version).build()
+        else ->
+          Health.down().withDetail("version", version).withDetail("status", Status.DOWN).build()
```

Comment:
One small thing, probably best to rename this to `PersonMatchScoreStatus` so we know exactly which is failing.